docs/learn-treasury.md

```diff
@@ -108,11 +108,13 @@ voting on it.
 Remember that the proposal has no metadata, so it's up to the proposer to create a description and
 purpose that the council could study and base their votes on.
 
-At this point, a Council member can turn this proposal into a motion to accept, or a motion to
-reject. It is possible (but unlikely) that a motion for both acceptance and rejection is created.
-After being turned into a council motion, the proposal needs 51% of the council to approve it to
-pass, or 51% to reject it to fail. If majority is not reached, the proposal remains in limbo -
-neither accepted nor rejected until more council members weigh in.
+At this point, a Council member can create a motion to accept or to reject the treasury proposal. It
```

Comment:
Hm it looks like there is a GH failure on one of the checks, the Substrate check-links one?